### PR TITLE
dev: New working deploy

### PR DESCRIPTION
**Pull request by xBalbinus**

### CODE REVIEW

File: development.json [hunk 1]
```diff
@@ -1,57 +1,11 @@
 {
   "version": 0,
   "networks": [
-    "arbitrumrinkeby",
     "evmostestnet",
```
Comment (anna-carroll) [inline development.json:4]: ahh okay i think requirements were not made sufficiently clear; i believe we want to re-deploy dev with goerli x2. so like, goerli-1 and goerli-2 as the two networks. no rinkeby (that's being sunsetted in a year, so we don't want to focus on it) and no evmos to begin with (priority on stability and ease of use)

File: archive/development/22-8-1-dev-fix/development.json [hunk 2]
```diff
@@ -0,0 +1,551 @@
+{
```
Comment (anna-carroll) [inline archive/development/22-8-1-dev-fix/development.json:1]: also commit the verification inputs which should be output as part of the deploy script whenever new contracts are deployed. those can be used to verify the contracts on etherscan

File: development.json (hunk 1 above)
Comment (xBalbinus) [reply development.json:4]: Understood - I think Conner just now also told me something similar so this is now clear.

File: archive/development/22-8-1-dev-fix/development.json (hunk 2 above)
Comment (xBalbinus) [reply archive/development/22-8-1-dev-fix/development.json:1]: Will recommit that with new deployment instances

**Pull request closed without merging**